api/app/controllers/application_controller.rb

```diff
@@ -1,2 +1,41 @@
 class ApplicationController < ActionController::API
+    before_action :authorized
+    def encode_token(payload)
+        JWT.encode(payload, 'my_s3cr3t')
```

Comment:
You can add the JWT secret key in an environment variable as opposed to having it in the code itself.